api/src/controllers/publications.js

```diff
@@ -161,26 +162,19 @@ async function getGoogleScholarPublications(req, res) {
     reachedEnd: false,
   };
 
-  const browser = await firefox.launch();
-  const context = await browser.newContext();
-  const page = await context.newPage();
-  await page.goto(url);
+  const page = await axios.get(url);
 
-  const resultLinks = await page.$$('.gsc_a_t a');
+  const $ = cheerio.load(page.data);
```

Comment:
I just followed the documentation but from [some googling](https://www.sitepoint.com/web-scraping-in-node-js/#:~:text=The%20result%20is%20assigned%20to%20the%20%24%20variable.%20The%20dollar%20sign%20was%20chosen%20because%20it%20is%20traditionally%20used%20in%20jQuery.), it looks like it represents a jQuery selector since `cheerio` is built on top of jQuery.